### PR TITLE
Pull request #7 from tenyiting/chapter7_redo_new

### PR DESCRIPTION
**Pull request by tenyiting**
Chapter7 redo new

### CODE REVIEW

**Comment by tenyiting** [comment]
ready

File: test/controllers/user_controller_test.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'test_helper'
```
Comment (ghost) [inline test/controllers/user_controller_test.rb]: please delete this file

File: db/schema.rb [hunk 2]
```diff
@@ -2,29 +2,23 @@
...
     t.string "email"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
     t.string "password_digest"
   end
 
 end
+ssssssssssssssssssssss
```
Comment (ghost) [inline db/schema.rb]: why have word in this file

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+    get 'home/index'
+    resources :posts
+    root 'static_pages#home', as: "home"
+    get 'static_pages/help', as: "help"
+
+    get "/signup", to: "users#new"
+    post "/signup", to: "users#create"
+    resources :users, only: %i(:new, :create, :show)
+  end
+end sssssssssssssss
```
Comment (ghost) [inline config/routes.rb]: ???

**Review by ghost — COMMENTED** [review]
cmt

File: config/routes.rb [hunk 4]
```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
-  resources :posts
-  get 'static_pages/home'
-  get 'static_pages/help'
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  scope "(:locale)", locale: /#{I18n.available_locales.join("|")}/ do
+    get 'home/index'
```
Comment (huongnt-2545) [inline config/routes.rb]: remove this line, you already defined at line 5

File: db/schema.rb (hunk 2 above)
Comment (tenyiting) [reply db/schema.rb]: sorry.. i was testing why the code not same in sublime.. this problem has no sssssssssss anymore

File: config/routes.rb (hunk 3 above)
Comment (tenyiting) [reply config/routes.rb]: sorry.. i was testing why the code not same in sublime.. this problem has no sssssssssss anymore

**Comment by tenyiting** [comment]
ready

**Review by huongnt-2545 — COMMENTED** [review]
Please create new commit for this pull request
NOT use commit merge from previous branch
```
# after fix comment
git add .
git commit -m "commit message"
...
```

File: config/routes.rb (hunk 4 above)
Comment (tenyiting) [reply config/routes.rb]: Screenshot from 2021-09-09 15-51-16

**Pull request closed without merging**